### PR TITLE
Foundry CircleCI 2.0 transition

### PR DESCRIPTION
**Pull request by khillyard**
New CircleCI 2.0 .circleci/config.yml configuration file created and tested.  This repo does not build as is since it is used as a template for new projects.  However, the new 2.0 configuration file build results match the 1.0 results.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "<server>",
+  "name": "foundry",
```
Comment (R-Campbell) [inline package.json]: I think this should be left as <server> as it's boilerplate that is meant to be modified.

File: .circleci/config.yml-converted [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# This configuration was automatically generated from a CircleCI 1.0 config.
```
Comment (R-Campbell) [inline .circleci/config.yml-converted]: I don't think we need the auto-generated file since we have the cleaned up version.

**Comment by khillyard** [comment]
Definitely agree with removing the auto-generated file and the original 1.0 circle.yml file.

Not sure about the "name" attribute in the package.json file.  I agree that it is useful to keep it as "<server>" for ease of use, but it will break the build.  If everyone is OK with breaking the build, I have no problem with the change.

**Comment by khillyard** [comment]
Closing since we are not using CircleCI

**Pull request closed without merging**